### PR TITLE
quickwhisper 1.0 (new cask)

### PR DESCRIPTION
**Pull request by agenkin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/q/quickwhisper.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cask "quickwhisper" do
+  version "1.0"
+  sha256 :no_check
+
+  url "https://f002.backblazeb2.com/file/quickwhisperapp/QuickWhisper.zip",
```
Comment (bevanjkay) [inline Casks/q/quickwhisper.rb:5]:
```suggestion
  url "https://quickwhisperapp.s3.us-west-002.backblazeb2.com/QuickWhisper_#{version}.zip"

File: Casks/q/quickwhisper.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      verified: "backblazeb2.com/file/quickwhisperapp/"
+  name "QuickWhisper"
+  desc "Fast and secure transcriptions"
+  homepage "https://quickwhisper.app/"
+
+  livecheck do
+    url "https://f002.backblazeb2.com/file/quickwhisperapp/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
```
Comment (bevanjkay) [inline Casks/q/quickwhisper.rb:14]:
```suggestion
  livecheck do
    url "https://f002.backblazeb2.com/file/quickwhisperapp/appcast.xml"
    regex(/QuickWhisper[._-]v?(\d+(?:\.\d+)+)\.zip/i)
    strategy :sparkle do |item, regex|
      item.url[regex, 1]
    end
  end
```

File: Casks/q/quickwhisper.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+cask "quickwhisper" do
+  version "1.0"
+  sha256 :no_check
+
+  url "https://f002.backblazeb2.com/file/quickwhisperapp/QuickWhisper.zip",
+      verified: "backblazeb2.com/file/quickwhisperapp/"
```
Comment (bevanjkay) [inline Casks/q/quickwhisper.rb:6]:
```suggestion
      verified: "quickwhisperapp.s3.us-west-002.backblazeb2.com/"

File: Casks/q/quickwhisper.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+cask "quickwhisper" do
+  version "1.0"
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/q/quickwhisper.rb:3]:
```suggestion
  version "1.0.35"
  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"

Comment (agenkin) [reply Casks/q/quickwhisper.rb]: Added latest version and sha256

**Comment by agenkin** [comment]
@bevanjkay your code suggestions are accepted. Thank you!

File: Casks/q/quickwhisper.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+cask "quickwhisper" do
+  version "1.0.35"
+  sha256 "bc8d1f18a32527e31e55e85a305b879ff77b09dcabe598f03bb6ab6ad798b544"
+
+  url "https://quickwhisperapp.s3.us-west-002.backblazeb2.com/QuickWhisper_#{version}.zip",
+      verified: "quickwhisperapp.s3.us-west-002.backblazeb2.com/"
+  name "QuickWhisper"
+  desc "Fast and secure transcriptions"
```
Comment (bevanjkay) [inline Casks/q/quickwhisper.rb:8]:
```suggestion
  desc "Audio transcription tool"

**Review by bevanjkay — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**